app/routes/agenda/minutes.js

```diff
@@ -47,7 +51,7 @@ export default class AgendaMinutesRoute extends Route {
       }
     }
     const meetingMinutes = await meeting.minutes;
-    return { minutes: meetingMinutes, mandatees, notas, announcements };
+    return { minutes: meetingMinutes, mandatees, notas, announcements, secretary };
```

Comment:
Fetching the secretary in the route isn't a good idea, because it can be updated without reloading the route.

- Click on "Inhoud updaten" of the meeting minutes of an agenda, observe that the current secretary is added
- Click on "Acties" > "Vergadering wijzigen"
- Update the secretary of the meeting
- Click on "Inhoud updaten" again, this time the secretary will not be updated
- Refresh the page
- Click on "Inhoud updaten", now the correct secretary is set

Instead of passing the secretary from the route, fetch the secretary when you're updating the contents of the meeting minutes, that way we're always using the "fresh" secretary.